### ISSUE TITLE
General Error (SQL on adding guild)

### ISSUE THREAD

**Issue by spitefultowel** [bbDKP/bbguild#271]
When attempting to add a guild with v2.0.0a6 the following is returned.

```
General Error
SQL ERROR [ mysqli ]

Column 'armoryresult' cannot be null [1048]

SQL

INSERT INTO phpbb_bb_guild (id, name, realm, region, battlegroup, roster, aion_legion_id, aion_server_id, level, players, achievementpoints, guildarmoryurl, emblemurl, game_id, min_armory, rec_status, guilddefault, armory_enabled, armoryresult, recruitforum, faction) VALUES (1, 'Common Affliction', 'Area 52', 'us', '', 0, 0, 0, 0, 0, 0, '', './../ext/bbdkp/bbguild/images/guildemblem/', 'wow', 50, 1, 1, 1, NULL, 0, 2)

BACKTRACE

FILE: (not given by php)
LINE: (not given by php)
CALL: msg_handler()

FILE: [ROOT]/phpbb/db/driver/driver.php
LINE: 855
CALL: trigger_error()

FILE: [ROOT]/phpbb/db/driver/mysqli.php
LINE: 193
CALL: phpbb\db\driver\driver->sql_error()

FILE: [ROOT]/phpbb/db/driver/factory.php
LINE: 329
CALL: phpbb\db\driver\mysqli->sql_query()

FILE: [ROOT]/ext/bbdkp/bbguild/model/player/guilds.php
LINE: 816
CALL: phpbb\db\driver\factory->sql_query()

FILE: [ROOT]/ext/bbdkp/bbguild/acp/guild_module.php
LINE: 323
CALL: bbdkp\bbguild\model\player\guilds->make_guild()

FILE: [ROOT]/ext/bbdkp/bbguild/acp/guild_module.php
LINE: 116
CALL: bbdkp\bbguild\acp\guild_module->AddGuild()

FILE: [ROOT]/includes/functions_module.php
LINE: 674
CALL: bbdkp\bbguild\acp\guild_module->main()

FILE: [ROOT]/adm/index.php
LINE: 81
CALL: p_master->load_active()
```

**Comment by spitefultowel** [comment]
Looks like the backend change is from armory_enabled to armoryresult so the front end needs to be corrected, or the backend does.

**Comment by dcJoker** [comment]
You can fix this temporarly by running following command in your database:
ALTER TBALE phpbb_bb_guild MODIFY armoryresult VARCHAR(255) NULL DEFAULT "";

However, I think this is not the right way. The next problem you will encounter is, that your Guildmembers can't open the BBGUILD Page inside of the UCP.

I think ist better we wait for the developer to make this whole think work again because its AWESOOOOME! :smile:

**Comment by Sajaki** [comment]
fixed in 03a16a018da60cc7508d4e3dcfa4b6930b9c77b8
